week-2-storage/README.md

```diff
@@ -209,4 +209,34 @@ kubectl port-forward service/mytenant-console 9090:9090 -n minio-tenant-1
 
 ## YAML way of deployment
 
-Please, see the reference: https://github.com/kubernetes/examples/tree/master/staging/storage/minio
\ No newline at end of file
+Please, see the reference: https://github.com/kubernetes/examples/tree/master/staging/storage/minio
+
+
+## Parallelization benchmarking
+
+### Data Set
+
+Data Set is available on the Kaggle [competition](https://www.kaggle.com/competitions/commonlitreadabilityprize/overview).
+
+### Data Set Integration
+
+Download `train.csv` files from Kaggle competition site and put the file in the `./dataset` folder.
+
+### Benchmarking Summary
+
+```
+Number of cpus                      : 8
+Dataset len                         : 2834
+
+[Single Process] Preprocessing      : 00:00:05
+[Single Process] Inference          : 00:01:56
+[Single Process] Total              : 00:02:02
+
+[Multiple Processes] Preprocessing  : 00:00:01
+[Multiple Processes] Inference      : 00:01:52
```

Comment:
looks like `Multiple Processes` is slower.

Reply:
any reasons why?

Reply:
Data processing has 3 steps:
- preporcessing (parallelization dropped down execution time from 5 to 1 second)
- tokenization (not paralleled, see comments blow)
- inference (not paralleled, see comments blow)

Because the experiment was run only once, the phenomenon of slower multiprocessing is just a fluctuation of observation. So, we should consider preprocessing parallelization stage only.